scripts/snyk.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# When packages change their dependency tree, we want to run snyk monitor. This
+# sends the updated dependency tree to snyk for monitoring.  We want to do this
+# when a commit has been made to master, not on PRs because the dependency tree
+# is likely still in flux while review is happening.
+#
+# For simplicity, this script just runs snyk monitor against all packages on
+# master commits, it doesn't try to figure out which packages were updated.
+
+if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_PULL_REQUEST" == "false" ]]; then
```

Comment:
Travis is throwing an error (and returning `exit 0`) with this line. Can you investigate?

Reply:
Show me where that error is? I only see the passing branch test: https://travis-ci.org/okta/okta-oidc-js/builds/406381991

This script technically should not have run yet, since we have not merged this PR to master?

Reply:
There is an error logged when attempting to evaluate the conditional. See [`L4891`](https://travis-ci.org/okta/okta-oidc-js/builds/406381991#L4891).

Reply:
Ah, thanks!  I just updated this script to be run with bash, looks like it's good now.